README.md

```diff
@@ -193,7 +193,7 @@ api.MetricsRetrieveLTV(&MetricsFilter{})
 api.MetricsListCustomerSubscriptions(&Cursor{}, "customerUUID")
 api.MetricsListCustomerActivities(&Cursor{}, "customerUUID")
 
-api.MetricsListActivities(&Cursor{PerPage: 100})
+api.MetricsListActivities(&cm.MetricsListActivitiesParams{StartDate: "2016-09-16T09:28:10Z", AnchorCursor: cm.AnchorCursor{PerPage: 5, StartAfter: "b45b1d3f-3823-424f-ab47-5a1d0c00a7f6"}})
```

Comment:
Including the cursor struct into the params struct results in the shortcut syntax where you can put the cursor struct fields into it directly:
```suggestion
api.MetricsListActivities(&cm.MetricsListActivitiesParams{StartDate: "2016-09-16T09:28:10Z", PerPage: 5, StartAfter: "b45b1d3f-3823-424f-ab47-5a1d0c00a7f6"})
```